### ISSUE TITLE
Add docs on deploying to Render

### ISSUE THREAD

**Issue by umar-ahmed** [NikitaVr/midna#11]
https://render.com/

**Comment by rohit-chaware** [comment]
@umar-ahmed I would like to work on this documentation task.

**Comment by umar-ahmed** [comment]
> @umar-ahmed I would like to work on this documentation task.

Hey @R4Rohit23 ! That's awesome.

The main thing you'll want to keep in mind is that the deployment really needs to do 4 things:

1. Set environment variables 
2. Build the Next.js app
3. Migrate the DB
4. Expose port for web traffic

You can take a look at the template we have for Railway to get an idea.

I haven't used Render myself, so I don't know if it requires a docker image and/or supports volume mounts for sqlite. Maybe you can start by researching that and responding back on this issue 🙂

Thanks again for volunteering!

**Comment by rohit-chaware** [comment]
@umar-ahmed I don't know about Database migration concepts. I know only about that the Render platform takes Environment variables and connect the pre deployed project on Github to be re-deployed for production.

**Comment by umar-ahmed** [comment]
> @umar-ahmed I don't know about Database migration concepts. I know only about that the Render platform takes Environment variables and connect the pre deployed project on Github to be re-deployed for production.

@R4Rohit23 Is there a way to run an arbitrary command in the terminal? Basically for the database migration, we just need to be able to run `yarn db:push` before starting the server

**Comment by rohit-chaware** [comment]
Yes We can do it in `package.json` file. To deploy any repo to render we need to pass a `build` command which is the starting point of any production level code. Like below code.

```
  "build": "npm install && npm install --prefix frontend && npm run build --prefix frontend"
```

**Comment by umar-ahmed** [comment]
Nice okay. We do something similar in Railway, but in the `start` command. So I think this shows Render deployment is viable.

@R4Rohit23 if you can focus on getting it deployed to Render, we can then create a template out of it
